azavea/raster-vision-website2
# Add in acquisition banner and change copy/links

**Overview**

This PR updates the references and links to Azavea to Element 84. It also adds a banner announcing the acquisition and linking to the blog post about it.

**Demo**
 _On desktop_
<img width="1425" alt="Screenshot 2023-03-16 at 2 34 49 PM" src="https://user-images.githubusercontent.com/77936689/225721752-d66df1a4-a826-4e10-842a-a6beef3f9e10.png">

_On mobile_
<img width="405" alt="Screenshot 2023-03-16 at 2 35 02 PM" src="https://user-images.githubusercontent.com/77936689/225721762-de1699c8-b8ce-4ffd-8a16-2e77523eaa42.png">

**Testing Instructions**
- Run `npm run compile-css` and then in another terminal tab, run `npm start` and go to localhost:8000
- Verify the banner looks like it does in [the designs](https://www.figma.com/file/dHNZK4IUu8W8QseGEkHBfY/RasterVision.IO-Website?node-id=1001%3A838&t=UhCbQKrFSJzD5QXe-4) and that the gradient and "learn more" link to the acquisition blog post
- Change the screen width and verify that it responds correctly
- Check that the footer now says “... framework brought to you by Element 84” and links to the E84 website
- Check the copyright says Element 84 instead of Azavea

Resolves #64
